pyemd/emd.pyx

```diff
@@ -91,6 +96,76 @@ def emd(np.ndarray[np.float64_t, ndim=1, mode="c"] first_histogram,
                                     distance_matrix, 
                                     extra_mass_penalty)
 
+def array_emd(first_array,
```

Comment:
I think this should be called `emd_samples` (if you can think of something better, I'm all ears), since the other `emd` functions also take arrays, and they all begin with `emd`.